### DISCUSSION TITLE
Stronger Rust code block integration

### DISCUSSION THREAD

**Discussion by TheNeikos** [srid/emanote#427] [Ideas]
This is less of a feature request, but more of a proposal + I would implement it with some guidance.

---

I would like to be able to do the following things with my Rust code blocks:

- Have them typechecked/run on build
- Have a link where readers can run the snippet on play.rust-lang.com
- Allow hiding lines starting with # (they still get filtered out for the previous steps)

I realize they are wildly different, but I didn’t want to make separate issues before there was any feedback.

Do these ideas sound interesting? (Potentially also for other languages) and if so, Id appreciate some pointers on how this could be added into the pipeline so I could try my luck at implementing them.

cheers!

**Comment by srid** [comment]
@TheNeikos All 3 of these can be done with https://emanote.srid.ca/guide/lua-filters right?
